src/test/unit/middlewares/authSocket.spec.js

```diff
@@ -7,16 +7,36 @@ describe('Auth socket midleware', () => {
   const error = createUnauthorizedError()
   const mockNextFunc = jest.fn()
 
-  it('Should call next function with UNAUTHORIZED error when access token is not given', () => {
-    const mockSocket = { handshake: { auth: {} } }
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+
+  it('Should call the next function with UNAUTHORIZED error when cookies are not given', () => {
+    const mockSocket = { request: { headers: {} } }
+
+    authSocketMiddleware(mockSocket, mockNextFunc)
+
+    expect(mockNextFunc).toHaveBeenCalledWith(error)
+  })
+
+  it('Should call the next function with UNAUTHORIZED error when access token is not given', () => {
+    const mockSocket = { request: { headers: { cookie: '' } } }
+
+    authSocketMiddleware(mockSocket, mockNextFunc)
+
+    expect(mockNextFunc).toHaveBeenCalledWith(error)
+  })
+
+  it('Should call the next function with UNAUTHORIZED error when access token is empty', () => {
+    const mockSocket = { request: { headers: { cookie: 'accessToken=;' } } }
 
     authSocketMiddleware(mockSocket, mockNextFunc)
 
     expect(mockNextFunc).toHaveBeenCalledWith(error)
   })
 
-  it('Should call next function with UNAUTHORIZED error when access token is not given', () => {
-    const mockSocket = { handshake: { auth: { token: 'invalid_token' } } }
+  it('Should call the nest function with UNAUTHORIZED error when access token is invalid', () => {
```

Comment:
small typo - 'nest'

In general, good idea to use cookies for make soccet connection more secure